go/vt/vttablet/tabletmanager/rpc_external_reparent.go

```diff
@@ -40,8 +43,18 @@ var (
 	finalizeReparentTimeout = flag.Duration("finalize_external_reparent_timeout", 30*time.Second, "Timeout for the finalize stage of a fast external reparent reconciliation.")
 
 	externalReparentStats = stats.NewTimings("ExternalReparents", "External reparenting time", "stage", "NewMasterVisible", "FullRebuild")
+
+	neverParent = flag.Bool("never_reparent", false, "if set to true, Vitess will never reparent. When running on a system that handles master/follower and replication, such as Amazon Aurora, Vitess should not try to reparent")
```

Comment:
It's not that Vitess should never reparent when running on Aurora it's that tablets that connect to the reader endpoint in Aurora can never safely become MASTER tablets as they never speak to a writer Aurora endpoint.

Reply:
`neverReparent`